pkg/apis/aws/v1alpha1/account_types.go

```diff
@@ -13,6 +13,9 @@ type AccountSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	AwsAccountID  string `json:"awsaccountid"`
+	IAMUserSecret string `json:"iamusersecret"`
+	Claim         string `json:"claim"`
```

Comment:
Can we make rename this to `ClaimLink` please

Reply:
updated